variationGenerator.js

```diff
@@ -5,7 +5,7 @@ export function generateVariations(stones) {
     const board = stonesToBoard(stones);
     const ladderedStone = getCords(stones.find(getIsLaddered));
 
-    return ladder(board, ladderedStone).map(position => boardToStones(position, ladderedStone));
+    return escapeLadder(board, ladderedStone).map(({position, winner}) => boardToStones(position, ladderedStone));

```

Comment:
There are some stuff the linter should have given you errors (like `winner` is an unused variable) :/...
I will go over the linting rules so please run `npm run lint` and fix everything afterwards